tools/ribodiff/ribodiff.xml

```diff
@@ -0,0 +1,157 @@
+<tool id="ribodiff" name="RiboDiff" version="0.2.2">
+    <description>Detect protein translational efficiency difference</description>
+    <requirements>
+        <requirement type="package" version="0.2.2">ribodiff</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[ 
+        TE.py 
+        -e $exp_design
+        -c $count_table
+        -o $ribodiff_out
+        -p 1
+        #if str($sumcntcutoff):
+            -s $sumcntcutoff
+        #end if
+        #if str($dispinitial):
+            -i $dispinitial
+        #end if
+        #if str($cutofffdr):
+            -q $cutofffdr
+        #end if
+        #if $dispdiff:
+            -d $dispdiff
+        #end if
+        #if $multitest:
+            -m $multitest_option
+        #end if
+        && mv *EmpDisp.scatter* EmpDisp.scatter.pdf
```

Comment:
It seems that these files could not be found. 
https://travis-ci.org/bgruening/galaxytools/jobs/334979039#L3381

Reply:
Yes, that's the problem with the pdf output currently. The tool doesn't generate any pdf files. Even when I do it with planemo. The Bioconda package works as expected. So, I can either remove the pdf feature for now and the tool will work. Or fix the pdf output problem. But I am stuck at that. Don't know what it could be.